Docs - Reorganize With Diataxis (#954)
iotaledger/iota.rs: documentation/docs/_partials/admonitions/_dust_protection.md

:::note Dust Protection
There is a [dust protection](https://wiki.iota.org/chrysalis-docs/faq#what-is-dust-protection-and-how-does-it-work)
mechanism implemented in the network protocol to prevent malicious actors from spamming the network to decrease node

> performance while keeping track of unspent amount (`UTXO`).

Reviewer: "unspent amount" does not abrreviate into UTXO

> to prevent malicious actors from spamming the network to decrease node performance while keeping track of unspent amount (`UTXO`).

Also this sentence doesn't make any sense. Actors spamming while keeping track of UTXO? They don't keep track of UTXO, nodes do. Is it supposed to be "while keeping their balance"? Is it supposed to be something like "...to decrease performance of nodes, which are keeping track of UTXO"?

Also, given that this is an admonition, you could make it just one sentence long, something like "Due to the dust protection(link), microtransaction below 1Mi can be sent to another address only if there are already at least 1Mi on that address"

> "... microtransactions below 1Mi of IOTA tokens can be sent to another address if there is already at least 1Mi on that address"

Reviewer: "1Mi" already means "One million of IOTA tokens".

> There is a [dust protection](https://wiki.iota.org/chrysalis-docs/faq#what-is-dust-protection-and-how-does-it-work)
> mechanism implemented in the network protocol to prevent malicious actors from spamming the network to decrease node
> performance while keeping track of unspent amount (`UTXO`).
> 
> 
> "... microtransactions below 1Mi of IOTA tokens can be sent to another address if there is already at least 1Mi on that address"

Author: ```suggestion
Due to the [dust protection](https://wiki.iota.org/chrysalis-docs/faq#what-is-dust-protection-and-how-does-it-work)
mechanism implemented in the network protocol, microtransactions below 1Mi of IOTA tokens can only be sent to another
address if there is already at least 1Mi on that address
```

> That's why the code in the example sent 1Mi, to comply with the protection.

Reviewer 2: ```suggestion
That's why the code in the example sent 1Mi, to comply with the protection.
```
```suggestion
That is why the code in the example sent 1Mi, to comply with the protection.
```
